pkg/hwevent/event.go

```diff
@@ -69,24 +69,21 @@ type Event struct {
 	// DataSchema - A link to the schema that the `Data` attribute adheres to.
 	// +optional
 	DataSchema *types.URI `json:"dataSchema,omitempty"`
-
-	Data *Data `json:"data,omitempty" `
 	// +required
+	Data *Data `json:"data,omitempty" `
```

Comment:
if this data is required field and you should remove omitempty

Reply:
updated